selenium_tests/course_info/page_objects/course_people_page_object.py

```diff
@@ -3,6 +3,8 @@
 from selenium.webdriver.support.wait import WebDriverWait
 
 from selenium_tests.course_info.page_objects.course_info_base_page_object \
+    import Locators as BasePageObjectLocators
```

Comment:
I know it's already unwieldy as all hell, but I saw this being used below and went looking for it in selenium common.  Maybe make it `CourseInfoBasePageObjectLocators`?  Or is that too much?